tests/lib/emails.js

```diff
@@ -104,7 +97,7 @@ define([
       });
 
       test('#recoveryEmailSecondaryEmailEnabled disabled for invalid email', function () {
-        return newVerifiedAccount('@featurenotenabledforthisdomain.com')
+        return newVerifiedAccount({domain: '@featurenotenabledforthisdomain.net'})
```

Comment:
I can't see any difference between calling `accountHelper.newVerifiedAccount` above and the local wrapper for it here. Can we delete the local wrapper and just call the helper method directly?